src/DocumentScope.zig

```diff
@@ -904,10 +887,6 @@ noinline fn walkContainerDecl(
     try scope.finalize();
 
     if (uses.items.len != 0) {
-        const scope_data = &context.doc_scope.scopes.items(.data)[@intFromEnum(scope.scope)];
-        scope_data.tag = .container_usingnamespace;
-        scope_data.data = .{ .container_usingnamespace = @intCast(context.doc_scope.extra.items.len) };
-
         try context.doc_scope.extra.ensureUnusedCapacity(allocator, uses.items.len + 2);
         context.doc_scope.extra.appendAssumeCapacity(@intFromEnum(node_idx));
         context.doc_scope.extra.appendAssumeCapacity(@intCast(uses.items.len));
```

Comment:
This is entire block should be removed including the `uses` variable that is declared further to the top.